Victor-Ugwueze/learn-git
# #1766255252 User signup page

### What does this PR do?
- Implements user sign up functionality

### Description of tasks to be completed?
- add signup form
- add validation

### How should this be manually tested?
- `git clone https://github.com/Victor-Ugwueze/learn-git`
- `git checkout ft-user-signup-1766255252`
- navigate to index.html
- you should see the signup form

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
[#1766255252](htttp://www.github.ccom, "link to story")

### Screenshots (if appropriate)
<img width="1303" alt="screen shot 2019-01-23 at 12 11 44 am" src="https://user-images.githubusercontent.com/24890605/51571797-81575e80-1ea3-11e9-9356-a6add5a27fe3.png">


### Questions
N/A
